vsts/scripts/dockerCleanup.sh

```diff
@@ -48,6 +48,7 @@ docker images
 #	b. oryxdevms/node-<major.minor>:latest
 #	b. oryxdevms/dotnetcore-<major.minor>:latest
 # - We should untag these images only after they have been pushed to a remote repository.
+UntagImages "test-*"
```

Comment:
should we also get rid of heroku images? how big are they?

Reply:
we're gonna need them for every test run, so performance will be better if we don't get rid of them